gamehype/forms.py

```diff
@@ -28,3 +36,21 @@ def validate_email(self, email):
         user = User.query.filter_by(email=email.data).first()
         if user is not None:
             raise ValidationError('Please use a different email address.')
+
+
+class AddGameForm(FlaskForm):
+    game_name = StringField(
+        'Game Title',
+        [validators.required(), validators.length(min=2, max=128)]
```

Comment:
this is weird? I'm not sure if its a byproduct of you trying to make the validator fire, but the format here is much different than every other field. They don't have `validators.required` etc`